frontend/src/metabase/entities/questions.js

```diff
@@ -42,6 +44,11 @@ const Questions = createEntity({
           Collections.actions.fetchList({ tree: true }, { reload: true }),
         );
 
+        const card = result?.payload?.question;
+        if (card) {
+          dispatch.action(API_UPDATE_QUESTION, card);
+        }
```

Comment:
If we don't update the card the selected collection won't be updated in the sidebar. Previously when we used `this.props.onSave(card);` it actually dispatch `API_UPDATE_QUESTION` under the hood.

![image](https://user-images.githubusercontent.com/1937582/177759892-f69cdcd4-04c4-4ad7-a0ee-e866a06320c4.png)